test/parallel/test-debugger-pid.js

```diff
@@ -50,5 +50,5 @@ interfacer.on('line', function(line) {
 });
 
 interfacer.on('exit', function(code, signal) {
-  assert.ok(code == 1, 'Got unexpected code: ' + code);
+  assert.strictEqual(code == 1, 'Got unexpected code: ' + code);
```

Comment:
I don't think this will work. `assert.ok()` asserts that the first argument is truthy. `assert.strictEqual()` asserts that the first argument is strictly equal to the second argument.